packages/main/src/plugin/index.ts

```diff
@@ -372,12 +372,12 @@ export class PluginSystem {
     const configurationRegistry = new ConfigurationRegistry(directories);
     configurationRegistry.init();
 
-    const telemetry = new Telemetry(configurationRegistry);
-    await telemetry.init();
-
     const proxy = new Proxy(configurationRegistry);
     await proxy.init();
 
+    const telemetry = new Telemetry(configurationRegistry);
```

Comment:
should we for example add a proxy parameter to Telemetry constructor ? just to ensure that we have a dependency and then nobody move the lines if the order is important

if the order is not important, it means we can keep the current/previous lines ?